cmd/copyEnumeratorInit.go

```diff
@@ -265,6 +268,39 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 			transfer.BlobTags = cca.blobTags
 		}
 
+		if cca.dryrunMode && shouldSendToSte {
+			glcm.Dryrun(func(format common.OutputFormat) string {
+				if format == common.EOutputFormat.Json() {
+					jsonOutput, err := json.Marshal(transfer)
+					common.PanicIfErr(err)
+					return string(jsonOutput)
+				} else {
+					if cca.fromTo.From() == common.ELocation.Local() {
+						// formatting from local source
+						return fmt.Sprintf("DRYRUN: copy %v%v to %v%v",
+							strings.Trim(cca.source.Value, "\\?"),
```

Comment:
These replacements may not be correct on all platforms, or situations.

E.g. C:\my\folder would come out as \C:\my\folder.

Consider using ToLongPath/FromLongPath in `common/LongPathHandler.go`.